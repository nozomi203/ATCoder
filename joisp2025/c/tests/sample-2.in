6 5 11
175 100 110 117 158
144 133 123 150 191
167 252 219 181 346
231 241 280 201 209
261 332 325 225 338
269 298 315 291 308
12
1 1 4 2
1 1 1 5
1 1 5 1
1 1 5 4
1 1 3 4
1 1 6 4
1 1 2 5
1 1 3 1
1 1 4 4
1 1 5 5
1 1 6 2
1 1 6 1
