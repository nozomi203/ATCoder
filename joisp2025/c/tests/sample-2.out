8
1
10
6
1
13
2
1
3
19
14
11
